lua/lexers/text.lua

```diff
@@ -1,4 +1,10 @@
 -- Copyright 2006-2020 Mitchell. See LICENSE.
 -- Text LPeg lexer.
 
-return require('lexer').new('text')
+local lexer = require('lexer')
+local lex = lexer.new('text')
+
+-- Whitespace
+lex.add_rule('whitespace', lexer.token(lexer.WHITESPACE, lexer.space^1))
```

Comment:
@moesasji it has to be `lex:add_rule` (a colon, not a dot).

Reply:
The joy of being far too used to using C based languages. Thanks for spotting this typo!